lib/systems/platforms.nix

```diff
@@ -471,6 +471,7 @@ rec {
       "x86_64-linux" = pc64;
       "armv5tel-linux" = sheevaplug;
       "armv6l-linux" = raspberrypi;
+      "armv7a-linux" = armv7l-hf-multiplatform;
```

Comment:
I just learned that the a stands for application: http://infocenter.arm.com/help/index.jsp?topic=/com.arm.doc.faqs/ka16827.html and `l` means little endian